Lib/test/support/__init__.py

```diff
@@ -2040,3 +2041,49 @@ def check_disallow_instantiation(testcase, tp, *args, **kwds):
         qualname = f"{name}"
     msg = f"cannot create '{re.escape(qualname)}' instances"
     testcase.assertRaisesRegex(TypeError, msg, tp, *args, **kwds)
+
+
+@contextlib.contextmanager
+def infinite_recursion(max_depth=75):
```

Comment:
This isn't in the original commit, FYI.

Reply:
Dang, good catch. Botched merge, somehow I didn't do this mistake for 3.9. GH-27785 is fine.